google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerImpl.java

```diff
@@ -2522,4 +2518,44 @@ Double get(double[] array, int i) {
       return array[i];
     }
   }
+
+  private static class StringArray extends AbstractList<String> implements Serializable {
+    private static final long serialVersionUID = 695127243179520960L;
+    private final List<com.google.protobuf.Value> underlying;
+
+    private StringArray(List<com.google.protobuf.Value> underlying) {
```

Comment:
nit: change this to ListValue so it is consistent with other classes in this file.